Cargo.lock

```diff
@@ -336,7 +333,7 @@ dependencies = [
  "lazy_static",
  "log",
  "num_cpus",
- "pdb",
+ "pdb 0.7.0",
```

Comment:
You could bump the pdb crate to 0.8.0 so we don't need two copies

Reply:
Oh, yes, that would be better. I briefly tried this but I ran into some issues where a size was updated from u16 to u64 and it wasn't clear where to do the truncation, so it wasn't a straightforward bump. I can take another look at it later.
Would you like to delay the symbolic bump until we've figured out the pdb bump, or are you ok with landing this first?

Reply:
No, just go ahead and we can do the pdb bump as a follow-up